packages/level-selector/src/public/level-selector.html

```
@@ -0,0 +1,10 @@
+<button type="button" data-level-selector-button>AUTO</button>
+<ul>
+  <% if (title) { %>
+  <li data-title><%= title %></li>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)